newrelic-api/src/main/java/com/newrelic/api/agent/MessageProduceParameters.java

```diff
@@ -128,6 +172,24 @@ public interface OutboundHeadersParameter {
 
     public interface Build {
 
+        /**
+         * Set the cloud provider's id for the message queue.
+         * This method is optional and can be bypassed by calling build directly.
+         */
+        Build cloudResourceId(String cloudResourceId);
+
+        /**
+         * Set the host name for the message queue.
+         * This method is optional and can be bypassed by calling build directly.
+         */
+        Build host(String host);
```

Comment:
DatabaseParameters has a single method for setting the host and port.
Do you think it makes sense to replicate that here?